### ISSUE TITLE
Support flutter

### ISSUE THREAD

**Issue by Panthro** [codecov/support#483]
Feature request to support flutter.io.

I tried using the dart implementation from the examples here https://github.com/codecov/dart but there are mismatch versions from flutter and the codecov dependency

**Comment by stevepeak** [comment]
Hi! We would love to support it. Should not be a problem. What we need are:

1. An example coverage report.
2. To build an example repository to show users how it works.

Can you help me with these items? Thanks!

**Comment by Panthro** [comment]
Yes sure, I already have a project setup with everything. It's also
reporting coverage to codacy, but it doesn't show in my dashboard probably
because it's ignoring dart as source code.

The project is /CrochetLand/app. Tomorrow I'll point you in the right
direction within the project to take a look, on he mobile now.

But if you get this before I have time to, take a look at the
travis-script.sh inside the device folder.

Cheers
